### DISCUSSION TITLE
CPU limit large export

### DISCUSSION THREAD

**Discussion by Gabriel416** [AntlerVC/firetable#1217] [Support / Q&A] [answered]
![Screen Shot 2023-04-19 at 8 54 53 PM](https://user-images.githubusercontent.com/23641060/233237909-dbdd3373-2be8-4dec-ae34-8d7eed9b8eb6.png)

Has anyone else run into this? If so how is there walk through on how to set up the cloud run settings correctly?

**Comment by bofeiw** [comment, marked as answer]
Hi, this may be an issue of the configuration we set when we deploy your large export cloud run job. We will investigate.

**Reply by jasonesg** [reply]
Hey @bofeiw, has this been solved out? I'm currently running into this issue as well

**Reply by bofeiw** [reply]
@jasonesg it should be fixed. What error are you having?

**Reply by jasonesg** [reply]
@bofeiw, I'm unable to export my table as it's stuck on 0 out of XXXXX. I've tried a smaller amount of cells as well and can't even export 200 rows. Not sure what the issue is.
![image](https://github.com/rowyio/rowy/assets/92497033/27708c11-2a60-4598-bb99-6dfa51629714)
